### PR TITLE
[NCL-7074] Remove build log pasing from BPM

### PR DESCRIPTION
**Pull request by janinko**
### Checklist:

* [ ] Have you added unit tests for your change?

### CODE REVIEW

File: build-coordinator/src/test/java/org/jboss/pnc/coordinator/test/DatastoreAdapterTest.java [hunk 1]
```diff
@@ -78,7 +77,6 @@ public void shouldStoreRepositoryManagerSuccessResult() throws DatastoreExceptio
         BuildRecord buildRecord = buildRecords.get(0);
 
         Assert.assertEquals(buildRecord.getStatus(), BuildStatus.SUCCESS);
-        Assert.assertTrue(buildRecord.getBuildLog().contains(BUILD_LOG));
         Assert.assertTrue(buildRecord.getBuildLog().contains(REPOSITORY_MANAGER_LOG));
```
Comment (matejonnet) [inline build-coordinator/src/test/java/org/jboss/pnc/coordinator/test/DatastoreAdapterTest.java:80]: Why did you leave this in ?

**Review by matejonnet — CHANGES_REQUESTED** [review]

**Comment by janinko** [comment]
retest this please

File: build-coordinator/src/test/java/org/jboss/pnc/coordinator/test/DatastoreAdapterTest.java (hunk 1 above)
Comment (janinko) [reply build-coordinator/src/test/java/org/jboss/pnc/coordinator/test/DatastoreAdapterTest.java]: Removed

**Comment by janinko** [comment]
This was done in https://github.com/project-ncl/pnc/pull/3987

**Pull request closed without merging**